tests/unit/monitor/data_monitoring/report/test_slack_report_summary_message_builder.py

```diff
@@ -52,35 +52,12 @@ def test_add_details_to_slack_alert_attachments_limit(test_results_summary):
     assert "The amount of results exceeded Slack" in attachments_as_string
 
 
-def test_owners_tags_and_subscribers_of_passed_tests_are_filtered_out(
```

Comment:
This test is no longer relevant because we removed the tags / owners / subscribers from the summary